src/serviceConnections/createSQLClient.js

```diff
@@ -0,0 +1,75 @@
+const knex = require('knex');
+
+const config = require('../config');
+const AWS = require('../utils/requireAWS');
+
+const getRDSEndpoint = async (rdsClient) => {
+  const clusterId = `aurora-cluster-${config.clusterEnv}`;
+
+  // Part of the endpoint's name is a random string generated by rds
+  // and we can't control it (it is always the same for accounts)
+  // so we can either:
+  // - store this random string in S3 or something and fetch it
+  // - or ask for an endpoint like I ended up doing here.
+  // https://forums.aws.amazon.com/thread.jspa?messageID=486170&#486199
+  // https://stackoverflow.com/questions/34990104/rds-endpoint-name-format
+
+  const { DBClusterEndpoints: endpoints } = await rdsClient.describeDBClusterEndpoints({
+    DBClusterIdentifier: clusterId,
+    Filters: [
+      { Name: 'db-cluster-endpoint-type', Values: ['writer'] },
+      { Name: 'db-cluster-endpoint-status', Values: ['available'] },
+    ],
+  }).promise();
+
+  if (endpoints.length === 0) {
+    throw new Error(`No available endpoints for ${clusterId}`);
+  }
+
+  return endpoints[0].Endpoint;
+};
+
+const getRDSParams = async () => {
+  const rdsClient = new AWS.RDS();
+  const endpoint = await getRDSEndpoint(rdsClient);
+  const username = 'api_role';
+
+  const signer = new AWS.RDS.Signer({
+    hostname: endpoint,
+    region: config.awsRegion,
+    port: 5432,
+    username,
+  });
+
+  const token = await signer.getAuthToken();
+
+  return {
+    host: endpoint,
+    port: 5432,
+    user: username,
+    password: token,
+    database: 'aurora_db',
+    ssl: { rejectUnauthorized: false },
```

Comment:
I would add the DB configuration files in the `default-config.js` in an object called `config` so you can just
do `user: config.db.username` for example.

Reply:
I think this fits better in its own config file rather than in default-config.js. For example, redis has its own folder with its own config.

I created the knexfile, when you can tell me your opinion on whether this would work.